packages/safe-apps-sdk/src/types/gateway.ts

```diff
@@ -1,233 +1,42 @@
-  toInfo?: AddressInfo;
-};
-
-export type Custom = BaseCustom & {
-  methodName: string | null;
-};
-
-export type MultiSend = BaseCustom & {
-  methodName: 'multiSend';
-  actionCount: number;
-};
-
-export type Creation = {
-  type: 'Creation';
-  creator: string;
-  transactionHash: string;
-  implementation: string | null;
-  factory: string | null;
-};
-
-export type TransactionStatus =
-  | 'AWAITING_CONFIRMATIONS'
-  | 'AWAITING_EXECUTION'
-  | 'CANCELLED'
-  | 'FAILED'
-  | 'SUCCESS'
-  | 'PENDING'
-  | 'PENDING_FAILED'
-  | 'WILL_BE_REPLACED';
-
-export type TransactionInfo = Transfer | SettingsChange | Custom | MultiSend | Creation;
-
-export type SafeAppInfo = {
-  name: string;
-  url: string;
-  logoUrl: string;
-};
-
-export type TransactionData = {
-  hexData: string | null;
-  dataDecoded: DataDecoded | null;
-  to: string;
-  value: string | null;
-  operation: Operation;
-};
-
-export type ModuleExecutionDetails = {
-  type: 'MODULE';
-  address: string;
-};
-
-export type MultiSigConfirmations = {
-  signer: string;
-  signature: string | null;
-};
-
-export type TokenType = 'ERC721' | 'ERC20' | 'NATIVE_TOKEN';
-
-export type TokenInfo = {
-  type: TokenType;
-  address: string;
-  decimals: number;
-  symbol: string;
-  name: string;
-  logoUri: string | null;
-};
-
-export type MultiSigExecutionDetails = {
-  type: 'MULTISIG';
-  submittedAt: number;
-  nonce: number;
-  safeTxGas: number;
-  baseGas: number;
-  gasPrice: string;
-  gasToken: string;
-  refundReceiver: string;
-  safeTxHash: string;
-  executor: string | null;
-  signers: string[];
-  confirmationsRequired: number;
-  confirmations: MultiSigConfirmations[];
-  gasTokenInfo: TokenInfo | null;
-};
-
-export type DetailedExecutionInfo = ModuleExecutionDetails | MultiSigExecutionDetails;
-
-export type GatewayTransactionDetails = {
-  executedAt: number | null;
-  txStatus: TransactionStatus;
-  txInfo: TransactionInfo;
-  txData: TransactionData | null;
-  detailedExecutionInfo: DetailedExecutionInfo | null;
-  txHash: string | null;
-  safeAppInfo: SafeAppInfo | null;
-};
+import { SafeBalanceResponse, TransactionDetails, TokenInfo } from '@gnosis.pm/safe-react-gateway-sdk';
+
+export {
+  AddOwner,
+  ChangeImplementation,
+  ChangeThreshold,
+  Creation,
+  Custom,
+  DataDecoded,
+  DetailedExecutionInfo,
+  DisableModule,
+  EnableModule,
+  Erc20Transfer,
+  Erc721Transfer,
+  InternalTransaction,
+  ModuleExecutionDetails,
+  MultiSend,
+  MultisigConfirmation,
+  MultisigExecutionDetails,
+  NativeCoinTransfer,
+  Operation,
+  Parameter,
+  RemoveOwner,
+  SafeAppInfo,
+  SetFallbackHandler,
+  SettingsChange,
+  SettingsInfo,
+  SwapOwner,
+  TokenInfo,
+  TokenType,
+  TransactionData,
+  TransactionInfo,
+  TransactionStatus,
+  Transfer,
+  TransferDirection,
+  TransferInfo,
+} from '@gnosis.pm/safe-react-gateway-sdk';
+
+export type GatewayTransactionDetails = TransactionDetails;
 
 export type TokenBalance = {
   tokenInfo: TokenInfo;
```

Comment:
TokenInfo is missing one parameter that I detected while working in this PR. Already opened a new PR to update it, will add the new sdk version here before merging

https://github.com/gnosis/safe-react-gateway-sdk/pull/58